crypto/dilithium/README.md

```diff
@@ -40,4 +43,34 @@ EVP_DigestSign(md_ctx.get(), signature, &signature_len, msg, msg_len);
 
 // Verify
 EVP_DigestVerify(md_ctx.get(), signature, signature_len, msg, msg_len);
-```
\ No newline at end of file
+```
+
+You can also sign Dilithium3 signatures using a `pem` private key file and the `bssl sign` tool. For example, to sign raw standard input you can use `echo "foo" | ./bssl sign -key private.pem`.
+
+## Benchmarks
+Benchmarks for KeyGen, Sign, and Verify functionality are available within the `bssl speed` tool, and can be explicitly filtered by running `./tool/bssl speed -filter Dilithium3`.
+
+Example output:
+```
+Did 4830 Dilithium3 keygen operations in 1014176us (4762.5 ops/sec)
+Did 2688 Dilithium3 signing operations in 1062613us (2529.6 ops/sec)
+Did 5124 Dilithium3 verify operations in 1014016us (5053.2 ops/sec)
+```
+
+Open Quantum Safe (OQS) provides benchmarks for the liboqs implementation of Dilithium based upon the same reference implementation. Full OQS benchmarks are available at [OQS benchmarking](https://openquantumsafe.org/benchmarking/visualization/speed_sig.html). For comparison we include the OQS Dilithium3 benchmarks from 2022-12-07.
+
+``` 
++-------------------------------+-----------+-----------------+---------+--------------+----------+----------------+
+|           Algorithm           | keypair/s | keypair(cycles) | sign/s  | sign(cycles) | verify/s | verify(cycles) |
++-------------------------------+-----------+-----------------+---------+--------------+----------+----------------+
+| Dilithium3 AWS-LC(x86_64-ref) |    4762.5 |          524934 |  2529.6 |       988298 |   5053.2 |         494736 |
+| Dilithium3 OQS (x86_64-ref)   |    5315.0 |          470266 |  1358.3 |      1840581 |   5690.7 |         439257 |
++-------------------------------+-----------+-----------------+---------+--------------+----------+----------------+
+```
+Dilithium3 AWS-LC(x86_64-ref) benchmarks were taken with the following configuration:
+```
+CPU	        Intel(R) Xeon(R) Platinum 8259CL CPU @ 2.50GHz 
+Platform 	Linux ip-172-31-41-0 5.15.0-1026-aws
+Instance	t3.medium
```

Comment:
You shouldn't benchmark on T* instances which have variable performance as a feature:
> Amazon EC2 allows you to choose between Fixed Performance instance families (e.g. M6, C6, and R6) and Burstable Performance Instance families (e.g. T3). Burstable Performance Instances provide a baseline level of CPU performance with the ability to burst above the baseline.
From https://aws.amazon.com/ec2/instance-types/

Reply:
Thanks, I was going to ask you before I spun one up if there was a good choice.  I wanted to target the same processor as OQS so the benchmarks were like-for-like. Looks like m5d.xlarge uses Intel(R) Xeon(R) Platinum 8259CL CPU @ 2.50GHz, what do you think of m5d?

Reply:
removed benchmark data in [096f384](https://github.com/awslabs/aws-lc/pull/724/commits/096f3840d3a43a74655b995a630e0544ae9b9afd)